tests/unittests/test_main.py

```diff
@@ -204,16 +204,34 @@ def test_load_game_by_name(self):
 	pass
 
 
-@skip_todo
-def test_load_game_by_path(self):
-	pass
+@mock.patch('horizons.main.start_singleplayer')
+def test_load_game_by_path(mock_start_singleplayer):
+	"""
+	Test that a specific savegame file given by path can be loaded from the command line.
+	An in-game map is used instead of an actual savegame file - see NOTE above.
```

Comment:
Ok, my NOTE might be confusing. It wasn't meant to be a reason why we're going to do the tests differently, but rather explain why the tests haven't been implemented yet.

In this case, we don't have to commit a savegame/map specifically for the tests, we should avoid that if possible. Here, we want to test that loading a file by a path is possible, so a temporary file will work just fine. Sketch:

```python
with tempfile.NamedTemporaryFile() as f:
  start_game("--load-game", f.name)
  ...
  assert options.game_identifier == f.name
```
(Note: you probably have to add the `suffix` argument to NamedTemporaryFile)

Any file will do for now, since in the code under test it isn't even inspected. If that code ever changes, the test will break and we can adapt it. But for now, a random empty file is fine.